src/view/commands/CommandResult.h

```diff
@@ -11,18 +11,24 @@
 #include <optional>
 
 class CommandResult {
```

Comment:
this class is a garbage can. it has everything. No good.

Reply:
Cleaned, Left only result enum, TaskId, and vector of tasks in it.